src/main/kotlin/network/warzone/mars/player/achievements/variants/RecordAchievement.kt

```diff
@@ -0,0 +1,81 @@
+package network.warzone.mars.player.achievements.variants
+
+import network.warzone.api.database.models.AgentParams
+import network.warzone.api.database.models.RecordType
+import network.warzone.mars.api.socket.models.PlayerUpdateData
+import network.warzone.mars.api.socket.models.PlayerUpdateEvent
+import network.warzone.mars.api.socket.models.PlayerUpdateReason
+import network.warzone.mars.player.achievements.AchievementAgent
+import network.warzone.mars.player.achievements.AchievementEmitter
+import org.bukkit.event.EventHandler
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+//TODO: This is currently experimental. I haven't done any tests on this achievement,
```

Comment:
Please ensure you've tested everything before merging

Reply:
The achievement here is currently unused, and I don't plan to use it in this release.